manifest.yml

```diff
@@ -12,3 +12,5 @@ applications:
   buildpack: liberty-for-java
   services:
   - sms_nlu_service
+env:
+  MODEL_ID: placeholder
```

Comment:
I think this is modelId in the code (checking).

Reply:
see above